### DISCUSSION TITLE
not-found not workin

### DISCUSSION THREAD

**Discussion by SySagar** [vercel/community#2929] [Help] [answered]
### Summary

I added a not-found.jsx file to my next 13 project. However on hitting unmatched routes instead of showing the custom 404 page , it keeps on throwing this same error:

` Error: Objects are not valid as a React child (found: object with keys {$$typeof, _payload, _init}). If you meant to render a collection of children, use an array instead.`

### Steps to Reproduce

*

**Comment by leerob** [comment, marked as answer]
Please open an issue (https://github.com/vercel/next.js) with a reproduction and we can look into this for you, thank you!
